libs/tx-gmail/src/auth.ts

```diff
@@ -22,13 +21,10 @@ export type OpenUrl = (url: string) => void;
  * @param {function} callback The callback to call with the authorized client.
  */
 export async function authorize(openurl?: OpenUrl) {
-  const credentialsPath = process.env.GMAIL_CREDENTIALS_PATH;
-  if (!credentialsPath || !existsSync(credentialsPath))
-    throw new Error("Cannot authorize gmail without credentials");
-  const credentials = JSON.parse(readFileSync(credentialsPath, "utf8"));
-  const { client_secret, client_id, redirect_uris } = credentials.installed
   const oAuth2Client = new google.auth.OAuth2(
```

Comment:
when this function is hit by the testing code, it doesn't matter what the credentials are because the google apis are mocked.  However, the test environment doesn't have the GMAIL_CREDENTIALS, so would throw for no good reason